Alexey/Fibonachi.py

```diff
@@ -0,0 +1,14 @@
+def main():
+    digit = int(input("Please type count of digits: "))

```

Comment:
It doesn't work
Task: Fibonacci Sequence - Enter a number and have the program generate the Fibonacci sequence to that number or to the Nth number.
Test:
Please type count of digits: 5
1
1
2
3
5
8
13
21
34
55
89
144
233
377
610
987
1597
2584
4181
6765
10946
17711
28657
46368
75025
Expected:
5

Test:
Please type count of digits: 2
1
1
2
3
5
8
13
21
34
55
89

Expected: 1